### PR TITLE
feat(cli/unstable): add `promptMultipleSelect()`

### PR DESCRIPTION
**Pull request by timreichen**

### CODE REVIEW

File: cli/unstable_prompt_multiple_select.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  Deno.stdout.writeSync(HIDE_CURSOR);
+  const buffer = new Uint8Array(4);
+  loop:
+  while (true) {
+    Deno.stdout.writeSync(encoder.encode(`${message}\r\n`));
+    for (const [index, value] of values.entries()) {
+      const selected = index === selectedIndex;
+      const start = selected ? INDICATOR : PADDING;
+      const checked = selectedIndexes.has(index);
+      const box = checked ? "☑️" : "☐";
```
Comment (kt3k) [inline cli/unstable_prompt_multiple_select.ts:57]: These characters seem unbalanced in terminal to me

Screenshot 2024-12-11 at 15 32 05

How about using `◯` and `◉` which are used by `npm:prompts` and `npm:@inquirer/prompts` in a similar context.

`npm:prompts`
Screenshot 2024-12-11 at 15 36 30

`npm:@inquirer/prompts`
Screenshot 2024-12-11 at 15 37 18

Comment (kt3k) [reply cli/unstable_prompt_multiple_select.ts:57]: Note: the code used for checking `npm:prompts` and `npm:@inquirer/prompts`

```js
import prompts from "npm:prompts";

await prompts({
  type: "multiselect",
  name: 'color',
  message: 'Pick colors',
  choices: [
    { title: 'Red', value: '#ff0000' },
    { title: 'Green', value: '#00ff00' },
    { title: 'Blue', value: '#0000ff' }
  ],
})
```

```js
import { checkbox, Separator } from "npm:@inquirer/prompts";

const answer = await checkbox({
  message: 'Select a package manager',
  choices: [
    { name: 'npm', value: 'npm' },
    { name: 'yarn', value: 'yarn' },
    new Separator(),
    { name: 'pnpm', value: 'pnpm', disabled: true },
    {
      name: 'pnpm',
      value: 'pnpm',
      disabled: '(pnpm is not available)',
    },
  ],
});
```

Comment (timreichen) [reply cli/unstable_prompt_multiple_select.ts]: Seems like the current checkboxes do not look the same in every terminal. I changed them.

**Review by kt3k — APPROVED** [review]
Looks good now 👍 

Screenshot 2024-12-12 at 13 15 36

**Pull request merged**